clang/lib/Headers/aiev2_scl2vec.h

```diff
@@ -521,13 +521,15 @@ broadcast_to_v32bfloat16(v4bfloat16 b) {
 
 INTRINSIC(v16accfloat)
 broadcast_to_v16accfloat(float b) {
-  return __builtin_aiev2_vbroadcastfloat_I512(b);
+  int as_int = *reinterpret_cast<int *>(&b);
```

Comment:
nit: I'd prefer `__builtin_bit_cast(int, b);`

Reply:
Same, I just forgot about it :/

Reply:
Updated